duckduckgo/macos-browser
# Use OpenSSL from DDG repo

Task/Issue URL: https://app.asana.com/0/0/1203990353723802/f

**Description**:
This change updates dependency on OpenSSL to our own repository where only binaries are stored.
This reduces OpenSSL dependency download size from ~1GB to 13MB, speeding up SPM dependencies
checkout and significantly reducing CI SPM cache size.

**Steps to test this PR**:
1. Verify that the app compiles and runs fine.
2. Verify that the app compiles in CI.
3. Smoke test Bitwarden integration (we’re using the exact same 1.1.1900 binary here without even recompiling it so there should be no difference, but just to be sure).


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
